lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -84,6 +89,16 @@ internal void PrepareDriverFolder(AppParameters appParameters, string driverFold
             // Add the appParameters into that folder structure
             _fileSets.AddJobFiles(appParameters);
 
+            // Add the reef-bridge-client jar to the global files
+            var reefBridgeClientJarFiles = Directory.GetFiles(JarFolder)
+                .Where(x => (!string.IsNullOrWhiteSpace(x)))
+                .Where(e => Path.GetFileName(e).ToLower().StartsWith(ClientConstants.ClientJarFilePrefix))
+                .ToList();
```

Comment:
In the manner of [JavaClientLauncher.cs](https://github.com/apache/reef/blob/3483e3d25a108934f118961dcf335410090b2d4b/lang/cs/Org.Apache.REEF.Client/Common/JavaClientLauncher.cs).

Reply:
Maybe add a comment to that effect?